projects/Mallard/src/hooks/use-issue-summary.tsx

```diff
@@ -83,13 +101,16 @@ const IssueSummaryProvider = ({ children }: { children: React.ReactNode }) => {
             }
         })
 
-        AppState.addEventListener('change', async appState => {
-            // when we foreground have another go at fetching again
-            if (appState === 'active') {
-                grabIssueAndSetLatest()
-            }
-        })
-    }, [issueId])
+        AppState.addEventListener(
```

Comment:
This event listener must be removed in the `useEffect` cleanup function, otherwise we're leaking progressively more and more event handlers during the execution of the app, I think. Which would mean duplicate calls to `grabIssueAndSetLatest`, and leaking memory.

Same for the `NetInfo` listener above, in fact :(

Reply:
This is an existing issue outside of the context of this PR?

Reply:
Sure, do you want to open a follow up task then?

Reply:
Created: https://trello.com/c/dtdqVf3K/1060-move-listeners-outside-of-useeffect-on-issue-summary